src/en/components/top-nav/base.md

```diff
@@ -14,4 +14,15 @@ A top navigation is a horizontal list of page links.
 {% docLinks locale stage figma github %}
 {% enddocLinks %}
 
-<br/>
+{% componentPreview "Top navigation component preview" %}
+<gcds-top-nav label="topbar" alignment="right" lang="en">
```

Comment:
Can we change the label to `Top navigation component preview` just to clearly label this is an example?

Also ca we switch the link and nav group text to match the anatomy images? Having very similar links to the actual navigation might be confusing.

Reply:
Good point about using the same text to match the anatomy images.
but for switching the label, wouldn't that create duplicate text for SR users? They would read "Top navigation component preview" twice in a row.